CMakeLists.txt

```diff
@@ -64,9 +84,57 @@ target_compile_definitions(${PROJECT_NAME} PUBLIC -DCFML_FACE_DETECTOR_MODEL_DIR
 set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 17)
 
 if(CLFML_FACE_DETECTOR_BUILD_EXAMPLE_PROJECTS)
-  add_executable(face_roi_demo ${CMAKE_CURRENT_LIST_DIR}/example/face_roi_demo/demo.cpp)
-  target_link_libraries(face_roi_demo PUBLIC face_detector)
+    if (CLFML_ROS2_PACKAGE_BUILD)
```

Comment:
Add here your other find_package statements, as they are only required for your example code:

```cmake
...
   if (CLFML_ROS2_PACKAGE_BUILD)
      find_package(ament_cmake REQUIRED)
    find_package(rclcpp REQUIRED)
    find_package(sensor_msgs REQUIRED)
    find_package(std_msgs REQUIRED)
    find_package(geometry_msgs REQUIRED)
    find_package(std_srvs REQUIRED)
  ... (rest of code)

```